clang/test/Driver/autocomplete.c

```diff
@@ -72,7 +72,7 @@
 // FVECLIBALL: Accelerate
 // FVECLIBALL-NEXT: MASSV
 // FVECLIBALL-NEXT: none
-// FVECLIBALL-NEXT: SVML
```

Comment:
Yes, I changed this test precisely because of the conditional compilation with `FLANG_LLVM_EXTENSIONS`.

If we merge `FLANG_LLVM_EXTENSIONS` with `USE_CLASSIC_FLANG`, then we could create a variant of this test, and distinguish between two with the `classic_flang` llvm-lit feature tag. However, it is also not very nice to have to maintain two variants of this test.

If llvm-lit supported conditional `RUN` lines, it would solve this problem. I could look into modifying the `RUN` lines to make `flang` indicate whether it is classic or not, *and then* choose the appropriate `FileCheck` command. This approach is quite hacky and requires additional changes to the driver, so it is not very appealing to me.

BTW I wasn't sure whether there are dependencies on `FLANG_LLVM_EXTENSIONS` that I'm not aware of. That's why I did not go ahead and merge the two macros into one.

Reply:
I have followed @peterwaller-arm's advice and merged the two macros into one `ENABLE_CLASSIC_FLANG`.